### PR TITLE
record a graph

### PR DESCRIPTION
**Pull request by chariga98**
@lilyclements this is ready for review

### CODE REVIEW

File: R/slopegraph_theme.R [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#' slopegraph_theme
```
Comment (lilyclements) [inline R/slopegraph_theme.R]: Can  you do a more meaningful title?

File: R/slopegraph_theme.R [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#' slopegraph theme
```
Comment (lilyclements) [inline R/slopegraph_theme.R:1]: Slopegraph theme

File: R/add_xy_area_range.R [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#' add xy area range
```
Comment (lilyclements) [inline R/add_xy_area_range.R:1]: Add*

**Comment by lilyclements** [comment]
@chariga98 looks good! Let me know when this is ready for re-review :)

File: R/add_xy_point_range.R [hunk 4]
```diff
@@ -1,4 +1,4 @@
-##' Create a File Path for XY Point Range Data
```
Comment (lilyclements) [inline R/add_xy_point_range.R:1]: @chariga98 how come you replaced this title for this function?

**Comment by chariga98** [comment]
@lilyclements I have just resolved the conflicts in my 'sub' branch you can now merge it. If there are some tasks that you want me to help you with am happy to help.

**Review by lilyclements — APPROVED** [review]

**Pull request merged**